docs/play/cdc-pay-integration.md

```diff
@@ -0,0 +1,151 @@
+# CDC Pay Integration
+This page illustrates some integration options for the CDC Pay module. The CDC Pay module can be intergated directly into your game development environment.
+
+## Pre-requisites
+
+- Crypto.com Pay Developer [access](https://merchant.crypto.com/users/sign_in/)
+- Optional:
+  - Webhook server (e.g. nodejs, [ngrok](https://ngrok.com/)) 
+
+### Overview
+The CDC Pay module makes use of REST API calls to trigger certain actions. These action will create events that are retrievable via webhooks. The following shell command illsutrates how to initiate the CDC Pay payment.
+
+```sh
+POST https://pay.crypto.com/api/payments
+```
+```sh
+curl https://pay.crypto.com/api/payments \
+  -u PUBLISHABLE_KEY: \
+  -d amount=2500 \
+  -d currency=USD \
+  -d description="Crypto.com Tee (Unisex)"
+```
+
+::: tip
+This API can be called using either your Secret Key or Publishable Key. These keys are generated in the merchant center [dashboard](https://merchant.crypto.com/users/sign_in/).
+:::
+
```

Comment:
This unfortunately does not work anymore in gitbook, you'd have to use this:

`{% hint style="info" %}
{% endhint %}`